compiler/test/formatter_outputs/comments.gr

```diff
@@ -3,15 +3,15 @@
  followed by a blank line
 */
 
-let myfun = (/* a */ x, /*post*/ y) => 10
+let myfun = (/* a */ x /*post*/, y) => 10
 
 /**
  * Then a doc
  */
 // Then a line
 let myfun1 = (/* a */ x, y) => 10
 
-let myfun2 = (x, /*post*/ y) => 10
+let myfun2 = (x /*post*/, y) => 10
```

Comment:
Why did you change the output locations of this?

Reply:
Because the comma isn't in the AST, the choice of where to place it in relation to the comments is arbitrary, and the new way of processing chooses the other.

Reply:
Ah, so it's actually *better* at keeping the input location now. Nice!